apps/src/templates/Button.jsx

```diff
@@ -68,7 +68,8 @@ class Button extends React.Component {
     tabIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     isPending: PropTypes.bool,
     pendingText: PropTypes.string,
-    __useDeprecatedTag: PropTypes.bool
+    __useDeprecatedTag: PropTypes.bool,
+    'aria-label': PropTypes.string
```

Comment:
Learned from this work that our Button component wasn't using the aria-label attribute –– I had assumed props were spread at the end, but they're not. Rather than spreading props into the `<Tag>` element (which could have unintended consequences), I explicitly passed the `aria-label` prop.

Reply:
I did check that an `aria-label` is also appropriate for a `div` and an anchor tag –– the two other Tag options in our Button component